Templates/HttpTrigger-CSharp/run.csx

```diff
@@ -1,28 +1,42 @@
+#r "Newtonsoft.Json"
+
 using System;
 using System.Net;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 public static Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
 {
-    var queryParamms = req.GetQueryNameValuePairs()
-        .ToDictionary(p => p.Key, p => p.Value, StringComparer.OrdinalIgnoreCase);
+    // parse query parameter
+    var queryParam = req.GetQueryNameValuePairs()
+                 .ToDictionary(p => p.Key, p => p.Value, StringComparer.OrdinalIgnoreCase);
 
     log.Verbose($"C# HTTP trigger function processed a request. RequestUri={req.RequestUri}");
 
+    // Get request body
+    string jsonContent = req.Content.ReadAsStringAsync().Result;
```

Comment:
Did you check that this doesn't blow up on GET requests? Not saying it does, but worth making sure